src/views/settings/Products.vue

```diff
@@ -0,0 +1,187 @@
+                      </v-col>
+                    </v-row>
+                  </v-form>
+                </v-container>
+              </v-card-text>
+              <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn color="blue darken-1" text @click="close">Annuleer</v-btn>
+                <v-btn color="blue darken-1" :disabled="!valid" text @click="save">Opslaan</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </v-toolbar>
+      </template>
+      <template v-slot:item.action="{ item }">
+        <v-icon small @click="deleteItem(item)">
+          mdi-delete
+        </v-icon>
+      </template>
+    </v-data-table> -->
+  </div>
+</template>
+
+<script>
+  import productsAPI from '@/services/ProductsAPI.js';
+  import productsTable from '@/components/products/Table';
+
+  export default {
+    data() {
+      return {
+        dialog: false,
+        loading: null,
+        valid: false,
+        errored: false,
+        errorMessage: '',
+        required: [
+          v => !!v || 'Dit veld is verplicht'
+        ],
+        headers: [
+          {text: 'Productnaam', align: 'left', sortable: true, value: 'name'},
+          {text: 'Type', align: 'left', sortable: true, value: 'type'},
+          {text: 'CNK', align: 'left', sortable: true, value: 'CNK'},
+          {text: 'Eenheid', align: 'left', sortable: false, value: 'outgoingUnit'},
+          {text: 'BTW', align: 'left', sortable: false, value: 'tax'},
+          {text: 'Verkoopsprijs', align: 'left', sortable: false, value: 'sellingPrice'},
+          {text: 'Verkoopsprijs/eenheid', align: 'left', sortable: false, value: 'sellingPricePerUnit'},
+          //{ text: 'supplementAdministration', align: 'left', sortable: false, value: 'supplementAdministration' },
+          //{ text: 'waitingTime', align: 'left', sortable: false, value: 'waitingTime' },
+          //{ text: 'unitSellingPrice', align: 'left', sortable: false, value: 'unitSellingPrice' },
+          //{ text: 'unitAdministrationPrice', align: 'left', sortable: false, value: 'unitAdministrationPrice' },
+          {text: 'Bewerken', align: 'right', sortable: false, value: 'action'},
+        ],
+        // te refactoren
+        /* types: ['Materiaal','Geneesmiddel','Voedingssupplement','Ontsmettingsmiddel'],
+        tax: ['6%','21'],
+        products: [],
+        editedIndex: -1,
+        editedItem: {
+          name: ''
+        },
+        defaultItem: {
+          name: ''
+        },*/
+      }
+    },
+    components: {
+      productsTable
+    },
+    mounted() {
+      this.loadProducts();
+    },
+    watch: {
+      dialog(val) {
+        val || this.close()
+      }
+    },
+    methods: {
+      async loadProducts() {
+        this.loading = true;
+        try {
+          const response = await productsAPI.getProduct();
+          this.products = response.data.products;
```

Comment:
object destructuring
```suggestion
          const { data: { products } } = await productsAPI.getProduct();
          this.products = products;
```